jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -523,23 +529,19 @@ private boolean isDDLCommand(int updatedCount, int columnCount) throws SQLExcept
       }
 
       if (character == '\\') {
-        antiSlash = true;
+        backslash = true;
       }
 
       if (character == '\'') {
-        if (antiSlash) {
-          antiSlash = false;
-        } else if (quoteString) {
```

Comment:
I see, I just checked on my local, reverted all of this, just removing the else part i.e. just had 

```
if (character == '\'') {
        if (antiSlash) {
          antiSlash = false;
        }
        if (quoteString) {
          quoteString = false;
        } else if (!doubleQuoteString) {
          quoteString = true;
        }
      }
```

And it works for all of these string; in the test case. What do you think?

```
String sqlQuery = "insert into test_table(id, name) values ('a', ';\"');" +
            "select * from test_table;" +
            "select * from test_table WHERE ID = \";'\";" +
            "select * from test_table WHERE ID = ';';" +
            "select '\\n', ';';" +
            "select replace('A\\;B', '\\', 'text');" +
            "select '\';"     +
            "select '\\', ';'"
```